### PR TITLE
DAH-1906 Add descriptive link text to edit buttons in application summary

### PR DESCRIPTION
**Pull request by tallulahkay**
[DAH-1906](https://sfgovdt.jira.com/browse/DAH-1906)

[DAH-1906]: https://sfgovdt.jira.com/browse/DAH-1906?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/assets/json/translations/react/en.json [hunk 1]
```diff
@@ -598,8 +598,16 @@
   "label.dontWantPreference": "I don't want this lottery preference",
   "label.downloadApplication": "Download Application",
   "label.downloadApplicationLang": "Download %{lang} Application",
+  "label.editAlternateContact": "Edit alternate contact information",
```
Comment (tallulahkay) [inline app/assets/json/translations/react/en.json:601]: not sure if I need these

Comment (chadbrokaw) [reply app/assets/json/translations/react/en.json:601]: I think we are safe to remove these. We can pull the translations over when we migrate to React

**Review by chadbrokaw — APPROVED** [review]

**Comment by jimlin-sfgov** [comment]
We should also push these new translation keys to Phrase.

File: app/assets/json/translations/locale-en.json [hunk 2]
```diff
@@ -671,7 +671,15 @@
             "done_with_this_address": "Done with this address",
             "dont_want_preference": "I don't want this lottery preference",
             "download_application": "Download {{lang}} Application",
+            "edit_alternate_contact": "Edit alternate contact information",
             "edit_eligibility": "Edit Eligibility",
+            "edit_household_details": "Edit household details",
+            "edit_household_members": "Edit household members",
+            "edit_income": "Edit income",
+            "edit_more_about_you": "Edit more about you",
+            "edit_preferences": "Edit preferences",
```
Comment (tallulahkay) [inline app/assets/json/translations/locale-en.json:680]: "Edit housing preferences"

File: app/assets/json/translations/locale-en.json [hunk 3]
```diff
@@ -671,7 +671,15 @@
             "done_with_this_address": "Done with this address",
             "dont_want_preference": "I don't want this lottery preference",
             "download_application": "Download {{lang}} Application",
+            "edit_alternate_contact": "Edit alternate contact information",
             "edit_eligibility": "Edit Eligibility",
+            "edit_household_details": "Edit household details",
+            "edit_household_members": "Edit household members",
+            "edit_income": "Edit income",
+            "edit_more_about_you": "Edit more about you",
```
Comment (tallulahkay) [inline app/assets/json/translations/locale-en.json:679]: "Edit demographics"

**Comment by tallulahkay** [comment]
Commented some language updates from Bridget

**Pull request merged**